### ISSUE TITLE
Shut down project, request to archive.

### ISSUE THREAD

**Issue by EvanCarroll** [kontalk/androidclient#1321]
Send a clear message please, and update the project or consider using GitHub functionality to archive it.

* Make a post here or on a new issue detailing why the project was abandoned, and whether or not someone can pick up the work here -- will you still accept pull requests? Do you want new people to step up as maintainers?
* If you will not be accepting and evaluating pull requests, please shut it down formally using the Github Project Archive functionality. It's better to tell people what caused the abandonment, but if you can't provide closure it's still better to not leave people hanging.

**Comment by daniele-athome** [comment]
You're right, I've been largely absent (life stuff and other projects). Kontalk is still in production but it won't last long - I promised myself to shut the Zeta server down by the end of the year but it'll probably last a few more months.

I'll make sure to make the necessary steps to warn people and provide closure.

**Comment by CrimsonFork** [comment]
So, now that the project has a sunset planned, any chance we could know what the "Kon" part of the project part is about? Or is it still private?

**Comment by opk12** [comment]
For reference, this is the F-droid ticket to request archiving the app, which is not getting releases since 2020, so that new users join a maintained service https://gitlab.com/fdroid/fdroiddata/-/issues/3628

**Comment by EvanCarroll** [comment]
> For reference, this is the F-droid ticket to request archiving the app, which is not getting releases since 2020, so that new users join a maintained service https://gitlab.com/fdroid/fdroiddata/-/issues/3628

Thanks for following up. I intended to do that.

**Comment by licaon-kter** [comment]
Archived: https://gitlab.com/fdroid/fdroiddata/-/commit/286a27285e21056be158c0894654699f8a4c24e6

**Comment by daniele-athome** [comment]
Thanks for archiving it on F-Droid! It's indeed time I announced the shutdown: https://forum.kontalk.org/t/project-and-server-shutdown/3714

I'm going to archive the repositories in a few days.
